src/Elastic.Apm.NLog/ApmServiceNameLayoutRenderer.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to Elasticsearch B.V under one or more agreements.
+// Elasticsearch B.V licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information
+
+using System.Text;
+using NLog;
+using NLog.Config;
+using NLog.LayoutRenderers;
+
+namespace Elastic.Apm.NLog
+{
+	[LayoutRenderer(Name)]
+	[ThreadSafe]
```

Comment:
Please also include `[ThreadAgnostic]`. See also https://github.com/NLog/NLog/wiki/How-to-write-a-custom-layout-renderer#how-to-optimize-performance-for-the-layout-renderer